run-tests.sh

```diff
@@ -7,5 +7,7 @@ $DIR/bin/console doctrine:database:drop --force
 $DIR/bin/console doctrine:database:create
 $DIR/bin/console doctrine:migrations:migrate --no-interaction
 sudo --preserve-env $DIR/bin/console elkarbackup:create_admin 
+mkdir /tmp/elkarbackup-tests
+mkdir /tmp/elkarbackup-tests/uploads
```

Comment:
You can use "mkdir -p /tmp/elkarbackup-tests/uploads" so that both are created if necessary.

Have you tested running tests twice in succession? Would that work OK without cleanning /tmp/elkarbackup-tests?

Reply:
Yes, it works ok